libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2521,6 +2523,15 @@ bool TypeChecker::visit(MemberAccess const& _memberAccess)
 	annotation.referencedDeclaration = possibleMembers.front().declaration;
 	annotation.type = possibleMembers.front().type;
 
+	if (
+		auto const& funType = dynamic_cast<FunctionType const*>(exprType) &&
```

Comment:
```suggestion
		auto const* funType = dynamic_cast<FunctionType const*>(exprType) &&
```

Reply:
Variable removed completely as not needed